pkg/kube/wait.go

```diff
@@ -146,6 +153,9 @@ func (w *waiter) waitForResources(created Result) error {
 				ok, err = w.podsReadyForObject(value.Namespace, value)
 			case *appsv1.ReplicaSet:
 				ok, err = w.podsReadyForObject(value.Namespace, value)
+			default:
+				// If the given type is not one we check for, continue in the loop
```

Comment:
is this needed? If I'm reading lines 55 and 160 correctly, `ok` should be `true` and `err` should be `nil` if we reach here, meaning that it'll continue iterating through the loop.

Reply:
I think you are right. I fixed this first and then moved to the next bug. Let me remove it and double check it works